lib/util/cache/repository/types.ts

```diff
@@ -10,6 +10,7 @@ export interface BaseBranchCache {
   configHash: string; // object hash of config
   extractionFingerprints: Record<string, string | undefined>; // matching manager fingerprints
   packageFiles: Record<string, PackageFile[]>; // extract result
+  onboardingSha: string | null;
```

Comment:
```suggestion
  onboardingSha?: string;
```

It shouldn't be a mandatory field for all the repos which are already onboarded